src/sonic-yang-models/tests/yang_model_tests/tests_config/bgp.json

```diff
@@ -1322,8 +1322,87 @@
                 ]
             }
         }
-    }
+    },
 
```

Comment:
I think, we should add YANG for BGP_PEER_RANGE table as well, please check and create the PR for review.